src/app/containers/NielsenAnalytics/Amp/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { string } from 'prop-types';
+
+// styled-components removes non-standard attributes (such as AMP attributes) on
+// server rendering. spreading props like this allows us to add AMP attributes
+// to the element.
+const AccessDiv = props => <div {...props} />;
+
+// Nielsen should only run in AU,
+// Using amp-geo to hide/display based on the country
+// amp-analytics doesn't generate any call if display none
```

Comment:
Be good to confirm this manually in testing, afaik the `display: none` does not display the element but leaves it in the DOM, the docs for `amp-geo` don't specifically call out `amp-analytics` for this behaviour https://amp.dev/documentation/components/amp-geo/
![image](https://user-images.githubusercontent.com/9645462/132640373-0ed70437-bb01-400b-964f-42df09192273.png)

Reply:
Tested